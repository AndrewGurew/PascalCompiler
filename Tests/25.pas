if x > 0 then modul := x else modul := -x;
if x>0 then modul:=x else modul:=-x;