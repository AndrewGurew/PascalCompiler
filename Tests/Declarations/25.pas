function foo(a:double; c:integer):integer;
begin
end

