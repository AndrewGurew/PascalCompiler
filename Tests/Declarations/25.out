Declaration
 ‚é¨foo - Function
 Params: a - Double, c - Integer
 Return type: Integer


